pages/common/ecpg.md

```diff
@@ -0,0 +1,28 @@
+# ecpg
+
+> Embedded SQL preprocessor for C programs.
+> More information: <https://www.postgresql.org/docs/current/app-ecpg.html>.
+
+- Preprocess a specific file:
+
+`ecpg {{prog1.pgc}}`
+
+- Preprocess from `stdin` and output to `stdout`:
+
+`ecpg -`
```

Comment:
This isn't a fully functioning command as there is nothing coming in from stdin.